### PR TITLE
[CS-2601] Add ability to send nfts in card wallet

### PR DESCRIPTION
**Pull request by oidoug**
### Description

Improves implementation of NFT transferring by checking valid interface for a token and applying the correct EIP, transfer method, and its parameters while creating a transaction.

Tokens were already being stored locally for faster loading and to use fewer contract calls, so I had to also add a new flag for rechecking them called `isInterfaceValidated`. On your first run, you'll see your token's names and interfaces logged in the console.

- [x] Completes #(CS-2601)

### Checklist

- [x] All UI changes have been tested on a small device

### Screenshots

https://user-images.githubusercontent.com/129619/153209934-00ff581d-be8d-469b-af51-ee5035af2ad7.mp4

### CODE REVIEW

File: src/handlers/web3.js [hunk 1]
```diff
@@ -117,6 +118,66 @@ export const toChecksumAddress = address => {
...
+
+  const lastBlockGasLimit = addBuffer(gasLimit.toString(), 0.9);
+  const paddedGas = addBuffer(
+    estimatedGas.toString(),
+    paddingFactor.toString()
+  );
+
+  // If the safe estimation is above the last block gas limit, use it
+  if (greaterThan(estimatedGas, lastBlockGasLimit)) {
+    logger.log('⛽ using orginal gas estimation', estimatedGas.toString());
```
Comment (danibonilha) [inline src/handlers/web3.js]: typo `original`

**Review by danibonilha — APPROVED** [review]
Nice job! 👏  Congrats!!  🚀

**Review by topwebtek7 — APPROVED** [review]
LGTM! tested on iOS and Android, worked well!

File: src/handlers/web3.js [hunk 2]
```diff
@@ -117,6 +118,66 @@ export const toChecksumAddress = address => {
...
+  addPadding = false,
+  paddingFactor = 1.1
+) => {
+  try {
+    const provider = await getEtherWeb3Provider();
+
+    const contract = new Contract(params.to, erc721ABI, provider);
+    const contractEstGas = await contract.estimateGas.transferFrom(
+      params.from,
+      params.to, // LOOK IT UP
```
Comment (lukemelia) [inline src/handlers/web3.js:167]: What does this comment mean? Should it be removed?

Comment (oidoug) [reply src/handlers/web3.js:167]: Yes, left that by mistake, I was considering changing the attr. name but `to` is the correct one in this case.

**Pull request merged**